src/main/java/seedu/address/logic/commands/RemarkCommand.java

```diff
@@ -32,7 +32,7 @@ public class RemarkCommand extends Command {
             + "Parameters: INDEX (must be a positive integer)\n"
             + "Example: " + COMMAND_WORD + " 1";
 
-    public static final String MESSAGE_NO_REMARK = "Remarks missing in entry.";
+    public static final String MESSAGE_NO_REMARK = "Oops! No remarks found in entry. Please try again!";
```

Comment:
try again for what

Reply:
its the standardised format for error msg